src/tests/all_paths.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <filesystem>
+#include <fstream>
+
+#include "algorithms/gfd/gfd.h"
+#include "parser/graph_parser/graph_parser.h"
+
+namespace tests {
+
+inline gfd::Gfd MakeGfd(std::filesystem::path const& gfd_path) {
+    std::ifstream f(gfd_path);
+    return parser::graph_parser::ReadGfd(f);
+};
```

Comment:
Maybe we should leave only the declaration here, and move the definition into the source file?